js/views/attachment_view.js

```diff
@@ -1,271 +1,290 @@
-        }
+      if (this.lightBoxView) {
+        this.lightBoxView.remove();
+      }
+      if (this.fileView) {
+        this.fileView.remove();
+      }
+      if (this.view) {
+        this.view.remove();
+      }
 
-        this.remove();
+      this.remove();
+    },
+    getFileType() {
+      switch (this.model.contentType) {
+        case 'video/quicktime': return 'mov';
+        default: return this.model.contentType.split('/')[1];
+      }
     },
-    getFileType: function() {
-        switch(this.model.contentType) {
-            case 'video/quicktime': return 'mov';
-            default: return this.model.contentType.split('/')[1];
-        }
-    },
-    onclick: function(e) {
-        if (this.isImage()) {
-            this.lightBoxView = new Whisper.LightboxView({ model: this });
-            this.lightBoxView.render();
-            this.lightBoxView.$el.appendTo(this.el);
-            this.lightBoxView.$el.trigger('show');
+    onclick() {
+      if (this.isImage()) {
+        this.lightBoxView = new Whisper.LightboxView({ model: this });
+        this.lightBoxView.render();
+        this.lightBoxView.$el.appendTo(this.el);
+        this.lightBoxView.$el.trigger('show');
+      } else {
+        this.saveFile();
+      }
+    },
+    isVoiceMessage() {
+      // eslint-disable-next-line no-bitwise
+      if (this.model.flags & textsecure.protobuf.AttachmentPointer.Flags.VOICE_MESSAGE) {
+        return true;
+      }
 
-        } else {
-            this.saveFile();
-        }
-    },
-    isVoiceMessage: function() {
-        if (this.model.flags & textsecure.protobuf.AttachmentPointer.Flags.VOICE_MESSAGE) {
-          return true;
-        }
+      // Support for android legacy voice messages
+      if (this.isAudio() && this.model.fileName === null) {
+        return true;
+      }
 
-        // Support for android legacy voice messages
-        if (this.isAudio() && this.model.fileName === null) {
-          return true;
-        }
-    },
-    isAudio: function() {
-        return this.model.contentType.startsWith('audio/');
-    },
-    isVideo: function() {
-        return this.model.contentType.startsWith('video/');
-    },
-    isImage: function() {
-        var type = this.model.contentType;
-        return type.startsWith('image/') && type !== 'image/tiff';
-    },
-    mediaType: function() {
-        if (this.isVoiceMessage()) {
-          return 'voice';
-        } else if (this.isAudio()) {
-          return 'audio';
-        } else if (this.isVideo()) {
-          return 'video';
-        } else if (this.isImage()) {
-          return 'image';
-        }
-    },
-    displayName: function() {
-        if (this.isVoiceMessage()) {
-            return i18n('voiceMessage');
-        }
-        if (this.model.fileName) {
-            return this.model.fileName;
-        }
-        if (this.isAudio() || this.isVideo()) {
-            return i18n('mediaMessage');
-        }
+      return false;
```

Comment:
⚠️ ESLint insisted on a return value so I added `return false` based on the existing code and the fact that implicit `return` is `returned undefined ~ return false`.